hs_core/views/__init__.py

```diff
@@ -1150,16 +1150,17 @@ def get_metadata_terms_page(request, *args, **kwargs):
 
 
 @login_required
-def get_user_data(request, user_id, is_group, *args, **kwargs):
+def get_user_or_group_data(request, user_or_group_id, is_group, *args, **kwargs):
     """
     This view function must be called as an AJAX call
 
-    :param user_id: id if the user for whom data is needed
+    :param user_or_group_id: id if the user for whom data is needed
```

Comment:
The param description I think should be: id of the user or group for which data is needed.